control_msgs/msg/MotionArgument.msg

```diff
@@ -0,0 +1,4 @@
+# Key Value pair to be used in MotionPrimitive messages
+
+string argument_name
+float64 argument_value
```

Comment:
```suggestion
string name
float64 value
```
Argument is kinda in the message type, does it need to repeat in the member variables too?. If everyone is okay with it, I'm fine with it

Reply:
@christophfroehlich @bmagyar any preferences?

Reply:
let's use the shorter names